guides/metrics.md

```diff
@@ -0,0 +1,54 @@
+| consensus\_latest\_block\_height           | Gauge     |                  | /status sync\_info number                                              |
+| consensus\_fast\_syncing                   | Gauge     |                  | Either 0 (not fast syncing) or 1 (syncing)                             |
+| consensus\_state\_syncing                  | Gauge     |                  | Either 0 (not state syncing) or 1 (syncing)                            |
+| consensus\_block\_size\_bytes              | Gauge     |                  | Block size in bytes                                                    |
+| consensus\_step\_duration                  | Histogram | step             | Histogram of durations for each step in the consensus protocol         |
+| consensus\_block\_gossip\_parts\_received  | Counter   | matches\_current | Number of block parts received by the node                             |
+| p2p\_message\_send\_bytes\_total           | Counter   | message\_type    | Number of bytes sent to all peers per message type                     |
+| p2p\_message\_receive\_bytes\_total        | Counter   | message\_type    | Number of bytes received from all peers per message type               |
+| p2p\_peers                                 | Gauge     |                  | Number of peers node's connected to                                    |
+| p2p\_peer\_receive\_bytes\_total           | Counter   | peer\_id, chID   | Number of bytes per channel received from a given peer                 |
+| p2p\_peer\_send\_bytes\_total              | Counter   | peer\_id, chID   | Number of bytes per channel sent to a given peer                       |
+| p2p\_peer\_pending\_send\_bytes            | Gauge     | peer\_id         | Number of pending bytes to be sent to a given peer                     |
+| p2p\_num\_txs                              | Gauge     | peer\_id         | Number of transactions submitted by each peer\_id                      |
+| p2p\_pending\_send\_bytes                  | Gauge     | peer\_id         | Amount of data pending to be sent to peer                              |
+| mempool\_size                              | Gauge     |                  | Number of uncommitted transactions                                     |
+| mempool\_tx\_size\_bytes                   | Histogram |                  | Transaction sizes in bytes                                             |
+| mempool\_failed\_txs                       | Counter   |                  | Number of failed transactions                                          |
+| mempool\_recheck\_times                    | Counter   |                  | Number of transactions rechecked in the mempool                        |
+| state\_block\_processing\_time             | Histogram |                  | Time between BeginBlock and EndBlock in ms                             |
```

Comment:
Is this list autogenerated? How can we ensure it's up to date?

Reply:
Will rewrite this to link to cometbft for default metrics and only document metrics exposed by rollkit specifically 👍🏽